app/controllers/repositories_controller.rb

```diff
@@ -356,6 +356,15 @@ def export_repositories
     end
   end
 
+  def export_repository_stock_items
+    if params[:row_ids]
+      RepositoryStockLedgerZipExport.generate_zip(params[:row_ids], current_user)
```

Comment:
how do you check that rows belong to specific repository and user can export them?